packages/web-api/src/WebClient.ts

```diff
@@ -714,9 +714,10 @@ function warnIfFallbackIsMissing(method: string, logger: Logger, options?: WebAP
     'Alternatively, you can provide an attachment-level `fallback` argument, though this is now considered a legacy field (see https://api.slack.com/reference/messaging/attachments#legacy_fields for more details).';
   if (isTargetMethod && typeof options === 'object') {
     if (isEmptyText(options)) {
```

Comment:
My intention of the new requirement is something like this:
```javascript
if (missingAttachmentFallbackDetected(options)) {
  if (isEmptyText(options)) {
    logger.warn(buildMissingTextWarning());
  }
  logger.warn(buildMissingFallbackWarning());
}
```

Reply:
i couldn't use that snippet ^ exactly because i think it broke existing functionality but i just pushed a commit that i think matches your original intention. could you give this another look when you get a chance?